src/libraries/System.Runtime/README.md

```diff
@@ -0,0 +1,18 @@
+# System.Runtime
+Contains fundamental classes and base classes that define commonly-used value and reference data types, events and event handlers, interfaces, attributes, and processing exceptions.
+
+Documentation can be found here: https://learn.microsoft.com/en-us/dotnet/api/system?view=net-7.0.
+
+TODO should we have a README for System.Private.CoreLib?
+
+## Contribution Bar
+- [x] [We consider new features, new APIs and performance changes](../../libraries/README.md#primary-bar)
+
+See the [Help Wanted](https://github.com/dotnet/runtime/issues?q=is%3Aissue+is%3Aopen+label%3Aarea-System.Runtime+label%3A%22help+wanted%22+) issues.
+
+## Source
+* [../../coreclr/System.Private.CoreLib/src/System](../../coreclr/System.Private.CoreLib/src/System)
+* Tests for this library live in [./tests](./tests). TODO do they live anywhere else?
```

Comment:
`src/libraries/System.Runtime/tests`, `src/libraries/System.Runtime.Extensions/tests`, and scattered about in a few other places as well.

This is the ref assembly that really gets split out across things

Reply:
Sounds good. It seems like we should add a README to System.Runtime.Extensions as well.

Reply:
Same with System.Numerics.Tensors and System.Numerics.Vectors